python/lsst/obs/decam/ingestCalibs.py

```diff
@@ -0,0 +1,54 @@
+from lsst.pipe.tasks.ingestCalibs import CalibsParseTask
+
+class DecamCalibsParseTask(CalibsParseTask):
+    def getInfo(self, filename):
+        """Get information about the image from the filename and/or its contents
+
+        @param filename: Name of file to inspect
+        @return File properties; list of file properties for each extension
+        """
+        phuInfo, infoList = CalibsParseTask.getInfo(self, filename)
+        # Single-extension fits without EXTNAME can be a valid CP calibration product
+        # Use info of primary header unit
+        if not infoList:
+            infoList.append(phuInfo)
+        return phuInfo, infoList
+
+    def translate_ccdnum(self, md):
+        """Return CCDNUM as a integer
+
+        @param md: FITS header metadata
+        """
+        if md.exists("CCDNUM"):
+            ccdnum = md.get("CCDNUM")
+        else:
+            self.log.warn("Unable to find value for CCDNUM")
+            ccdnum = None
+        # Some MasterCal from NOAO Archive has 2 CCDNUM keys in each HDU
+        # Make sure only one integer is returned. 
+        if not isinstance(ccdnum, int) and isinstance(ccdnum, (tuple, list)):
```

Comment:
It's arguably more Pythonic to check for integral types and sequences rather than an explicit int or tuple/list:
```python
import numbers
import collections
if not isinstance(ccdnum, numbers.Integral) and isinstance(ccdnum, collections.Sequence):
```